src/GlideConversion.php

```diff
@@ -18,13 +20,45 @@
     /** @var string */
     private $conversionResult = null;
 
-    public static function create(string $inputImage): self
+    /** @var string */
+    private $temporaryDirectory = null;
+
+    public static function create(string $inputImage, $config = []): self
+    {
+        return new self($inputImage, $config);
+    }
+
+    public function setTemporaryDirectory($config)
```

Comment:
Could this function be simplified by always having a `temporaryDirectory` set? In the constructor you could set the one we are using now as a default.